android/quest/build.gradle.kts

```diff
@@ -171,6 +171,13 @@ android {
       dimension = "apps"
       applicationIdSuffix = ".ecbis"
       versionNameSuffix = "-ecbis"
+      manifestPlaceholders["appLabel"] = "MOH eCBIS Preview"
+    }
+
+    create("ecbis_liberia") {
+      dimension = "apps"
+      applicationIdSuffix = ".ecbis_liberia"
+      versionNameSuffix = "-ecbis_liberia"
       manifestPlaceholders["appLabel"] = "MOH eCBIS"
```

Comment:
ah so we're going to make the current one preview and have the new one be production? I think we can put "production" in the name instead of "liberia" for clarity

Reply:
I think we should do the reverse since the production app is what the client will interact with even on _Playstore_ and should be concise, that is 
- For prod - the flavor should be `ecbis` and the suffix `.ecbis`
- For preview - the flavor should be `ecbis_preview` and the suffix `.ecbis_preview`

Reply:
If we are unable to enforce the integrity for a fresh install then I suppose we can follow that approach. We will however need a new _Playstore_ release APK upload and verification process for the new application id cc @pld

Reply:
That's a nice catch. Does the mean we need to de-list the current app?

Reply:
k so then we'd have 2 releases on play store the prod and preview one? I'm fine w/that, I'd worry a bit that if we have to de-list the current app, we take the risk of delay on getting up a new app